crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -544,30 +556,40 @@ module sui::validator_set {
     }
 
     public(friend) fun is_duplicate_validator(validators: &vector<Validator>, new_validator: &Validator): bool {
+        count_duplicates_vec(validators, new_validator) > 0
+    }
+
+    fun count_duplicates_vec(validators: &vector<Validator>, new_validator: &Validator): u64 {
```

Comment:
What is the rationale for counting duplicates vs just erroring on a single duplicate?

Reply:
I need a way to check for duplicate when `new_validator` may already be part of `validators` (e.g., when we change only the network pk, 'new_validator' might already be pending, and then I expect to have exactly 1 duplicate in active/pending validators).

in any case, I'll rename `new_validator` to `validator`